include/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -252,7 +252,7 @@ SECTIONS
 	. = ALIGN(_region_min_align);
 	_image_ram_start = .;
 
-#if defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
+#if defined(CONFIG_SOC_SERIES_STM32F0X) && defined(CONFIG_SRAM_VECTOR_TABLE)
```

Comment:
Since CONFIG_SRAM_VECTOR_TABLE depends on F0, we don't need the CONFIG_SOC_SERIES_STM32F0X check anymore, right ?

Reply:
This change is only present to preserve bisectability. This chunk gets moved into a linker snippet in a later commit